app/src/main/res/values-land/dimens.xml

```diff
@@ -135,4 +129,14 @@
   <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
   <dimen name="feedback_item_question_view_margin_end">64dp</dimen>
   <dimen name="feedback_item_question_view_margin_bottom">8dp</dimen>
+
+  <!-- Interaction Item: Exploration View -->
+  <dimen name="interaction_item_exploration_view_margin_start">80dp</dimen>
+  <dimen name="interaction_item_exploration_view_margin_top">24dp</dimen>
+  <dimen name="interaction_item_exploration_view_margin_end">48dp</dimen>
+
+  <!-- Interaction Item: Question View -->
+  <dimen name="interaction_item_question_view_margin_start">64dp</dimen>
+  <dimen name="interaction_item_question_view_margin_top">24dp</dimen>
```

Comment:
confirming this 32 or 24 or it's like 24+8 = 32?
https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/208ac665-ca91-48c9-87f6-e7aa7ed3ac29/